kind/kind.sh

```diff
@@ -48,42 +41,10 @@ function _wait_kind_up {
     kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
 }
 
-function _fetch_kind() {
-    mkdir -p "${KIND_DIR}"
-    KIND="${KIND_DIR}"/.kind
-    if [ -f "$KIND" ]; then
-        current_kind_version=$($KIND --version | awk '{print $3}')
-    fi
-    if [[ $current_kind_version != $KIND_VERSION ]]; then
-        echo "Downloading kind v$KIND_VERSION"
-        if [[ "$OSTYPE" == "darwin"* ]]; then
-            curl -LSs https://github.com/kubernetes-sigs/kind/releases/download/v$KIND_VERSION/kind-darwin-${ARCH} -o "$KIND"
-        else
-            curl -LSs https://github.com/kubernetes-sigs/kind/releases/download/v$KIND_VERSION/kind-linux-${ARCH} -o "$KIND"
-        fi
-        chmod +x "$KIND"
-    fi
-}
-
-function _prepare_config() {
-    echo "Building manifests..."
-
-    cp $KIND_MANIFESTS_DIR/kind.yml "${KIND_DIR}"/kind.yml
-    sed "s/$kind_registry_name/${KIND_REGISTRY_NAME}/g" "${KIND_DIR}"/kind.yml > "${KIND_DIR}"/kind.yml.tmp && mv "${KIND_DIR}"/kind.yml.tmp "${KIND_DIR}"/kind.yml
```

Comment:
sed has different behaviors on mac and linux. :-)

Reply:
as I don't want to block mac user use this repo to setup their own dev cluster at local.
but... it seems the changes here is about do we need flexibility on port number?

Reply:
Exactly, why not we fix the content of both kind.yml and local-registry.yml :)